ProcessLib/DirichletBc.h

```diff
@@ -12,19 +12,15 @@
 
 #include <vector>
 
-#include "MeshLib/MeshSearch/NodeSearch.h"
+#include "NumLib/IndexValueVector.h"
 
 namespace ProcessLib
 {
 
 /// A dirichlet boundary condition is represented by a list of global indices
 /// with corresponding values.
 template <typename IndexType>
-struct DirichletBc final
-{
-    std::vector<IndexType> global_ids;
-    std::vector<double> values;
-};
+using DirichletBc = NumLib::IndexValueVector<IndexType>;
```

Comment:
I personally don't care it. If you don't like it, may I ask you to do it in another PR?